python-avocado.spec

```diff
@@ -26,28 +26,13 @@
 # enabled by default.
 %global with_tests 1
 
-%if 0%{?rhel}
-%global with_python3 0
-%else
-%global with_python3 1
-%endif
-
 # Python 3 version of Fabric package is new starting with Fedora 29
-%if %{with_python3} && 0%{?fedora} >= 29
+%if 0%{?fedora} >= 29
 %global with_python3_fabric 1
 %else
 %global with_python3_fabric 0
 %endif
 
-# Python2 binary packages are being removed
-# See https://fedoraproject.org/wiki/Changes/Mass_Python_2_Package_Removal
-# python2-resultsdb_api package has been removed in F30
-%if (0%{?fedora} && 0%{?fedora} <= 29) || (0%{?rhel} && 0%{?rhel} <= 7)
-%global with_python2_resultsdb 1
-%else
-%global with_python2_resultsdb 0
-%endif
-
 # The Python dependencies are already tracked by the python2
```

Comment:
Is this comment still accurate now that python2 has been ropped?

Reply:
Yes, because while we're dropping Python 2, Fedora still supports it.  This is where this comes from:

https://fedoraproject.org/wiki/Packaging:AutoProvidesAndRequiresFiltering#Filtering_provides_and_requires_after_scanning